susemanager/src/mgr_bootstrap_data.py

```diff
@@ -754,6 +754,32 @@
     "venv-salt-minion",
 ]
 
+PKGLISTDEBIAN11 = [
+    # gnupg dependencies
```

Comment:
On a minimal Debian11 installation from ISO, with only the core system installed (no "modules", marked at tasksel), installing `gnupg` requires the following:
```
Reading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  dirmngr gnupg-l10n gnupg-utils gpg gpg-agent gpg-wks-client gpg-wks-server
  gpgconf gpgsm libassuan0 libksba8 libldap-2.4-2 libldap-common libnpth0
  libsasl2-2 libsasl2-modules libsasl2-modules-db libsqlite3-0 pinentry-curses
Suggested packages:
  dbus-user-session libpam-systemd pinentry-gnome3 tor parcimonie xloadimage
  scdaemon libsasl2-modules-gssapi-mit | libsasl2-modules-gssapi-heimdal
  libsasl2-modules-ldap libsasl2-modules-otp libsasl2-modules-sql pinentry-doc
The following NEW packages will be installed:
  dirmngr gnupg gnupg-l10n gnupg-utils gpg gpg-agent gpg-wks-client
  gpg-wks-server gpgconf gpgsm libassuan0 libksba8 libldap-2.4-2
  libldap-common libnpth0 libsasl2-2 libsasl2-modules libsasl2-modules-db
  libsqlite3-0 pinentry-curses
0 upgraded, 20 newly installed, 0 to remove and 0 not upgraded.
Need to get 0 B/9,064 kB of archives.
After this operation, 18.5 MB of additional disk space will be used.
Do you want to continue? [Y/n]
```
So basically we now have the bundle, gnupg, and all gnupg dependencies that are not hart of the core system.